### PR TITLE
version: v3.1.0

### LINKED ISSUE #850: release v3.1.0
# Release checklist

- [x] Ensure all [milestoned issues/PRs](https://github.com/mandiant/capa/milestones) are addressed, or reassign to a new milestone.
  - [x] https://github.com/mandiant/capa/milestone/14
- [x] Add the `dont merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/mandiant/capa/pulls) and [capa-rules](https://github.com/mandiant/capa-rules/pulls).
- [x] Ensure the [CI workflow succeeds in master](https://github.com/mandiant/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [ ] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples`  offenses are allowed in the nursery).
- [x] Review changes
  - capa https://github.com/mandiant/capa/compare/v3.0.3...master
  - capa-rules https://github.com/mandiant/capa-rules/compare/v3.0.3...master
- [x] Update [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections: see template below
  - Update `Raw diffs` links
  - Create placeholder for `master (unreleased)` section
    ```
    ## master (unreleased)

    ### New Features

    ### Breaking Changes

    ### New Rules (0)

    -

    ### Bug Fixes

    ### capa explorer IDA Pro plugin

    ### Development

    ### Raw diffs
    - [capa <release>...master](https://github.com/mandiant/capa/compare/<release>...master)
    - [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/<release>...master)
    ```
- [x] Update [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py)
- [x] Create a PR with the updated [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/mandiant/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md).
- [ ] Verify GH actions [upload artifacts](https://github.com/mandiant/capa/releases), [publish to PyPI](https://pypi.org/project/flare-capa) and [create a tag in capa rules](https://github.com/mandiant/capa-rules/tags) upon completion.
- [ ] [Spread the word](https://twitter.com)
- [ ] Update internal service

### PR DESCRIPTION
**Pull request by williballenthin**
closes #850

### CODE REVIEW

File: capa/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "3.0.3"
+__version__ = "3.10.0"
```
Comment (mr-tz) [inline capa/version.py:1]:
```suggestion
__version__ = "3.1.0"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,17 +2,46 @@
...
+
+### Bug Fixes
+
+### capa explorer IDA Pro plugin
+
+### Development
+
+### Raw diffs
+- [capa <release>...master](https://github.com/mandiant/capa/compare/v3.1.0...master)
+- [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/v3.1.0...master)
```
Comment (mr-tz) [inline CHANGELOG.md:21]:
```suggestion
- [capa v3.1.0...master](https://github.com/mandiant/capa/compare/v3.1.0...master)
- [capa-rules v3.1.0...master](https://github.com/mandiant/capa-rules/compare/v3.1.0...master)
```

Comment (mr-tz) [reply CHANGELOG.md:21]: maybe we update the template to use general text

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,17 +2,46 @@
...
+
+## v3.1.0 (2022-01-10)
+This release improves the performance of capa while also adding 23 new rules and many code quality enhancements. We profiled capa's CPU usage and optimized the way that it matches rules, such as by short circuiting when appropriate. According to our testing, the matching phase is approximately 66% faster than v3.0.3! We also added support for Python 3.10, aarch64 builds, and additional MAEC metadata in the rule headers.
+  
+This release adds 23 new rules, including nine by Jakub Jozwiak of Mandiant. @ryantxu1 and @dzbeck updated the ATT&CK and MBC mappings for many rules. Thank you!
+  
+And as always, welcome first time contributors!
+
+  - @kn0wl3dge
+  - @jtothej
```
Comment (mr-tz) [inline CHANGELOG.md:31]: `@cl30` as well?

File: CHANGELOG.md [hunk 4]
```diff
@@ -61,8 +89,8 @@
 - show features script: add backend flag #430 @kn0wl3dge
 
 ### Raw diffs
-- [capa <release>...master](https://github.com/mandiant/capa/compare/v3.0.3...master)
-- [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/v3.0.3...master)
+- [capa <release>...master](https://github.com/mandiant/capa/compare/v3.0.3...v3.1.0)
+- [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/v3.0.3...v3.1.0)
```
Comment (mr-tz) [inline CHANGELOG.md:93]:
```suggestion
- [capa v3.0.3...v3.1.0](https://github.com/mandiant/capa/compare/v3.0.3...v3.1.0)
- [capa-rules v3.0.3...v3.1.0](https://github.com/mandiant/capa-rules/compare/v3.0.3...v3.1.0)
```

Comment (mr-tz) [reply CHANGELOG.md:93]: update to general text?

**Comment by mr-tz** [comment]
thanks, looks good besides the minor suggested changes

**Review by mike-hunhoff — APPROVED** [review]
thank you! lgtm 🚀

**Pull request merged**